pkgs/development/python-modules/sip/default.nix

```diff
@@ -15,19 +16,20 @@
 
 buildPythonPackage rec {
   pname = "sip";
-  version = "6.8.3";
+  version = "6.8.6";
   pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-iIVHsBi7JMNq3tUZ6T0+UT1MaqC6VbfMGv+9Rc8Qdiw=";
+    hash = "sha256-f8lZ5I5uxdWvi9Am9p9eJNCLPLirs0IXb1q4AwzAfXo=";
   };
 
   nativeBuildInputs = [ setuptools ];
 
   propagatedBuildInputs = [
     packaging
     setuptools
+    setuptools-scm
   ] ++ lib.optionals (pythonOlder "3.11") [ tomli ];
 
   # There aren't tests
```

Comment:
Not true.

Reply:
dupe of https://github.com/NixOS/nixpkgs/pull/347606#discussion_r1794401301

Reply:
How is that comment related to the existence of tests?

Reply:
ah sorry, thought it was a comment on the line addition

Reply:
Regarding non-existance of tests can't look into it right now UnU will leave that unresolved